source/MaterialXGenMdl/mdl/materialx/stdlib_1_6.mdl

```diff
@@ -4426,6 +4426,71 @@ export float4 mx_noise2d_float4(
     return float4(v.x, v.y, v.z, w) * mxp_amplitude + mxp_pivot;
 }
 
+export color mx_noise2d_color3(
```

Comment:
It looks like these MDL-specific definitions for type variations of noise nodes were also replaced by language-independent graphs in a pull request for MaterialX 1.38.8, so we don't need to restore them to the codebase!

https://github.com/AcademySoftwareFoundation/MaterialX/pull/1653

Reply:
😓
Thanks, looks like we have not been synced our repo for a while. The extra noise functions are removed.